content/en/ecosystem/distributions.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Distributions
+description:
+  Libraries, services, and apps with first-party support for OpenTelemetry.
+aliases: [/distributions]
+---
+
+OpenTelemetry [distributions](/docs/concepts/distributions/) are a way of
+customizing OpenTelemetry components so that they're easier to deploy and use
+with specific observability back ends.
+
+Any third-party can customize OpenTelemetry components with back-end, vendor, or
+end-user specific changes. You don't have to use a distributions in order to use
+OpenTelemetry components, though distributions might facilitate usage under
+certain circumstances, such as specific vendor requirements.
+
+The following list contains a sample of OpenTelemetry distributions and the
+component they customize.
```

Comment:
```suggestion
The list below contains a sample of OpenTelemetry distributions and the
component they customize. Note that those are third-party projects and are in no way validated or endorsed by the OpenTelemetry community.
```

Reply:
Edited your suggestion to change "below" to "following" as per https://developers.google.com/style/word-list#below